passive/PassiveLinkDef.h

```diff
@@ -28,13 +27,13 @@
 #pragma link C++ class R3BVacVesselCool+;
 #pragma link C++ class R3BCave+;
 #pragma link C++ class R3BNeutronWindowAndSomeAir+;
-#pragma link C++ class R3BGeoCave;
+#pragma link C++ class R3BGeoCave+;
 #pragma link C++ class R3BGeoGDML+;
-#pragma link C++ class R3BGeoPipe;
-#pragma link C++ class R3BGeoVacVesselCool;
-#pragma link C++ class R3BGeoTarget;
-#pragma link C++ class R3BGeoMagnet;
-#pragma link C++ class R3BGeoPassivePar;
-#pragma link C++ class R3BPassiveContFact;
+#pragma link C++ class R3BGeoPipe+;
+#pragma link C++ class R3BGeoVacVesselCool+;
+#pragma link C++ class R3BGeoTarget+;
+#pragma link C++ class R3BGeoMagnet+;
+#pragma link C++ class R3BGeoPassivePar+;
+#pragma link C++ class R3BPassiveContFact+;
```

Comment:
Is there a special reason to add ROOT streamers to Geo classes? RuntimeDB should take care about I/O of those objects.